<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundEffectForm</class>
 <widget class="QWidget" name="SoundEffectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxRepeat">
        <property name="text">
         <string>Repeat every</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputInterval">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSeconds">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSoundEffectName">
        <property name="text">
         <string>sound effect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonPlayPause">
        <property name="text">
         <string>II</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVolume">
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputVolume">
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
